client/application.cpp

```diff
@@ -1223,14 +1232,26 @@ void application::initialize()
 		pico_face_tracker = xr_session.create_pico_face_tracker();
 	}
 
-	if (fb_body_tracking_supported)
+	if (utils::contains_all(xr_extensions, std::array{XR_FB_BODY_TRACKING_EXTENSION_NAME, XR_META_BODY_TRACKING_FULL_BODY_EXTENSION_NAME, XR_META_BODY_TRACKING_FIDELITY_EXTENSION_NAME}))
+	{
+		XrSystemBodyTrackingPropertiesFB fb_body_properties = xr_system_id.fb_body_tracking_properties();
+		spdlog::info("    FB body tracking support: {}", (bool)fb_body_properties.supportsBodyTracking);
+		if (fb_body_properties.supportsBodyTracking)
+			body_tracker = xr_session.create_fb_body_tracker();
+	}
+
+	if (body_tracker.index() == 0 && utils::contains_all(xr_extensions, std::array{XR_HTC_PATH_ENUMERATION_EXTENSION_NAME, XR_HTC_VIVE_XR_TRACKER_INTERACTION_EXTENSION_NAME}))
 	{
-		fb_body_tracker = xr_session.create_fb_body_tracker();
+		spdlog::info("    HTC body tracking support: true");
+		body_tracker = xr_session.create_htc_body_tracker();
 	}
 
-	if (pico_body_tracking_supported)
+	if (body_tracker.index() == 0 && utils::contains(xr_extensions, XR_BD_BODY_TRACKING_EXTENSION_NAME))
```

Comment:
Prefer `std::holds_alternative<pico_body_tracker>` so that it doesn't depend on variant ordering